### ISSUE TITLE
Flexispot EF1

### ISSUE THREAD

**Issue by 2bias2u** [iMicknl/LoctekMotion_IoT#50]
For flexispot EF1 the Desk model: Flexispot E5B works.
I have the Flexispot EF1 und I used the table from the E5B and it is working.

**Comment by omarrha44** [comment]
Yes indeed it is working for EF1 model, using E5B template. You just need to delete M switch because there is no M button on EF1 but there is preset 4 so you just need to add preset 4 from getting uart data from main page.

**Comment by sonar98** [comment]
I have got two EF1 desks. I bought them at different times.
My first desk has a HCB103A-2 controller/remote.
The second desk has a HCB213A-1.

Using the E5B wiring table it works with the HCB213A-1 but it doesn't work with the other desk.

**Comment by ThomDietrich** [comment]
Thanks @lukes-ha, this is good news.

This ticket might be linked to https://github.com/iMicknl/LoctekMotion_IoT/issues/37 btw
